Add scripts to use Android App Bundles

jtreanor opened the pull request:
This adds two scripts script, `tools/build-app-bundle.sh` and `tools/install-app-bundle.sh` which makes it easier to build and test Android App Bundle (.aab) files.

The scripts are simplified versions of the ones in WPAndroid. We should think about sharing these at some point.

Related to https://github.com/wordpress-mobile/WordPress-Android/pull/9734.

To test:

- Build an Android App Bundle: `./tools/build-app-bundle.sh`. This will prompt you to install `bundletool` if you don't have it installed.
- Install it in a device or emulator: `./tools/install-app-bundle.sh artifacts/wcandroid-1.8-rc-1.aab`. 
- Check that it works correctly (particularly Google login).
- Install the APK: `adb install artifacts/wcandroid-1.8-rc-1-universal.apk`.
- Check that it works correctly (particularly Google login).

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

jtreanor commented:
That's strange @loremattei 🤔 I had tested Google login and . it worked fine. Certainly worth investigating this.

loremattei commented:
@jtreanor  I investigated this a little bit more and I think the problem is with the signature. If you sign the generated apk with the proper key, Google login works. If you don't pass any key to `bundletool` it uses a _debug_ key and google login doesn't work (at least on my test device).

jtreanor commented:
Ah, thanks @loremattei. I suspected that might be the case. I'll update the script with signing.

jtreanor commented:
@loremattei This is ready for another pass now. Here is what I have changed:

- Added `./tools/build-app-bundle.sh` which builds the app bundle and extracts the universal APK.
- Updated `./tools/install-app-bundle.sh` to correctly sign to app. Confirmed Google login works now.

The scripts are adapted from WPAndroid.

loremattei commented:
Cool! I tested the changes @jtreanor and it looks like we can ship it :+1:
